devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -111,10 +111,16 @@ def parse(String description) {
 
 def off() {
     zigbee.off()
+    if (device.getDataValue("model") == "HY0105") {
+	zigbee.command(zigbee.ONOFF_CLUSTER, 0x00, "", [destEndpoint: 0x02])

```

Comment:
You need to return all the commands in a list. As written this command will be the only one executed and `zigbee.off()` will not be executed. Rather than sending two commands I think you could instead send a single command to endpoint 0xFF. That should turn off all the endpoints that support the OnOff cluster.